packages/phone-number-privacy/combiner/src/server.ts

```diff
@@ -87,3 +88,41 @@ export function createHandler<R extends OdisRequest>(
 ): PromiseHandler<R> {
   return meteringHandler(catchErrorHandler(enabled ? handler : disabledHandler<R>))
 }
+
+export function startProxy(req: any, res: any, config: CombinerConfig) {
+  const proxy = httpProxy.createProxyServer({})
```

Comment:
Maybe we can pass a timeout here just in case. Something a little bigger than the v2 or something, but that would avoid blindly relaying that the v2 is having good timeouts